config/services.yaml

```diff
@@ -18,3 +18,5 @@ services:
         exclude:
             - "../src/Bundle/*"
             - "../src/Exception/*"
+
+    Doctrine\Persistence\ManagerRegistry: '@doctrine'
```

Comment:
Why is this needed? It might break the app that is using this bundle

Reply:
without this the autowire does not work well with some versions :/ 

https://travis-ci.org/KnpLabs/DoctrineBehaviors/jobs/638475665?utm_medium=notification&utm_source=github_status

Reply:
I think it should be compatible with both. 
Moving alias to a services that's there in some conditions might break stuff.

Reply:
Standalone test case for ManagerRegistry with own would definitelly help

Reply:
Sorry but I don't understand what you mean
 imo autowiring should not be used and doctrine service explicitely injected

Reply:
ok
apologize i though doctrine bundle wasn't a dev dependency
i found the weird alias stuff -- with if (false) condition ?? -- and type hinted on it, everything seems ok